example/runtime/pytorch-cn-mirror/requirements.txt

```diff
@@ -9,19 +9,14 @@ pycocotools
 opencv-python-headless
 # fixed versions
 numpy==1.21.6; python_version=='3.7'
-torch==1.11.0; python_version=='3.7'
-torchaudio==0.11.0; python_version=='3.7'
-torchdata==0.3.0; python_version=='3.7'
-torchtext==0.12.0; python_version=='3.7'
-torchvision==0.12.0; python_version=='3.7'
 scikit-learn==1.0.2; python_version=='3.7'
 scipy==1.7.3; python_version=='3.7'
 numpy==1.23.2; python_version>='3.8'
-torch==1.12.1; python_version>='3.8'
+torch==1.12.1; python_version>='3.7'
 scikit-learn==1.1.1; python_version>='3.8'
 scipy==1.8.1; python_version>='3.8'
 importlib-metadata>=4.0.0, <=4.2.0;python_version < '3.8'
-torchaudio==0.12.1; python_version>='3.8'
-torchdata==0.4.1; python_version>='3.8'
-torchtext==0.13.1; python_version>='3.8'
-torchvision==0.13.1; python_version>='3.8'
+torchaudio==0.12.1; python_version>='3.7'
+torchdata==0.4.1; python_version>='3.7'
+torchtext==0.13.1; python_version>='3.7'
+torchvision==0.13.1; python_version>='3.7'
```

Comment:
The version of python will always >= 3.7, so we can remove the conditions?

Reply:
> The version of python will always >= 3.7, so we can remove the conditions?

OK